docs/MODEL_CONFIGURATION.md

```diff
@@ -0,0 +1,77 @@
+[Back to *Chat with your data* README](../README.md)
```

Comment:
You weren't to know this, but we are trying to get all the docs filenames to be lowercase and less shouty! Are you able to amend this new one to save us in the future?

Reply:
I tried doing this in the dev container using vscode and it wouldn't let me. I'll try again using the web interface to see if it can do better.

Reply:
Done.